docs/networking/networking-basics.md

```diff
@@ -0,0 +1,520 @@
+# Networking Basics
+
+> Written by Dan Ford and originally published on <a href="https://dlford.io/computer-networking-binary-hex-explained/" target="_blank" rel="noopener noreferrer">dlford.io</a>.<br />
+> Copyright &copy; 2019 dlford.io all rights reserved.
+
+This article is written with the intent of providing a basic understanding of computer networking and IP addressing concepts, which require a basic understanding of binary and hex. I've done my best to make it easy to follow for somebody with no experience on this topic. 
+
+_Note: I use the term "modem" very loosely in this article because most consumer modems these days have a built in switch and routing functions. You should be aware that technically a modem does not do routing, in only modulates a signal out to your ISP. In the context of this article I refer to a modem as a modem/router combo unit, when I say "router or modem", I am referring to whichever routing device your system is directly connected to._
+
+## Basic Network Settings
+
+### Discover Your Network
+
+In the context of a home network, you most likely have a modem or router handling DHCP, DNS, and routing for your network. The first thing you need to do is find the IP address of your router or modem on the network. An IP address is a string of numbers separated by decimal points, that identifies each computer on a network, and must therefore be unique to each computer on the same network.
+
+- Windows
+  - Right-click the start button, and choose Windows PowerShell
+  - Type the command `ipconfig | findstr /i "Gateway"`
+- Mac
+  - Go to Applications > Utilities, and double click on Terminal
+  - Type the command `netstat -nr | grep default`
+- Linux
+  - Open a terminal shell, the method for this varies by distribution, on Ubuntu you can use the hotkey `control` + `shift` + `t`
+  - Type the command `ip route | grep default`
+
+You should get a value of something similar to `192.168.0.1`, this is the IP address of your gateway. Open a web browser and put this number in the address bar with `http://` in front of it, for example `http://192.168.0.1`
+
```

Comment:
Some router will only admit calls in https and will return an error if called with http. It's an edge case, I now, however I should be incorporated here.